archtec/scripts/common.lua

```diff
@@ -41,6 +42,15 @@ function core.kick_player(player_name, reason)
 	return core.disconnect_player(player_name, reason)
 end
 
+function core.kick_inactive_player(player_name)
+	if archtec.is_online(player_name) then
+		minetest.chat_send_all(minetest.colorize("#FF0000", player_name .. " got kicked due to inactivity."))
```

Comment:
Still undecided if color should be changed in-game. I suppose, red is fine for accent, being easily seen and all.